doc/configuration/network.md

```diff
@@ -86,12 +86,14 @@ Use the CA certificate with `jcli`.
     from. This is the public address that will be distributed to other peers
     of the network that may find interest into participating to the blockchain
     dissemination with the node.  Currently only TCP is supported.
-- `public_id`: (optional) This is a static identifier, 24 bytes encoded in hexadecimal. They are used
-  to bootstrap the connection to the node if the node introduce itself as a trusted peer.
-  **Most of the user don't need to set this value** and in fact we are working toward potentially
-  removing the need for this value.
-- `listen_address`: (optional) [multiaddr][multiaddr] specifies the address the node
-    will listen to to receive p2p connection. Can be left empty and the node will listen
+- `node_key_file`: (optional) Path to a file containing a bech32-encoded ed25519 secret key.
+  The keys are used to advertize the node in network gossip and to authenticate
+  a connection to the node if the node is used as a trusted peer.
+  **Most of the users don't need to set this value** as the key will be randomly
```

Comment:
Will it be unique for each jormungandr run, or will it be generated and saved on the first run?

Reply:
That is yet to be decided, as it requires some thought on how we want to handle identification. I think both choices make sense, probably a node that wants to serve as a trusted peer would want to reuse the same key, while 'casual' nodes can use this as an ephemeral key, but again I haven't thought this through. Identification / Verification of identity is out of scope for this PR (but I still had to add handling of ids on the topology side since they are required by poldercast)

Reply:
Well, I think keeling this key ephemeral when not provided by a user is fine. Let them generate a key by hand if they want it to be persistent.